src/Octoshift/ReclaimService.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Octoshift.Models;
+using OctoshiftCLI;
+using OctoshiftCLI.Models;
+
+[assembly: InternalsVisibleTo("OctoshiftCLI.Tests")]
+namespace Octoshift
+{
+    public class ReclaimService
+    {
+        private readonly GithubApi _githubApi;
+        private readonly OctoLogger _log;
+
+        public const string CSVHEADER = "mannequin-user,mannequin-id,target-user";
+
+        internal class Mannequins
```

Comment:
why `internal` and not `private`?  if the intention is to mock it in tests then this this needs to be 100% injected and as a separate service.